### ISSUE TITLE
\frontmatter/\mainmatter ignored when counting pages

### ISSUE THREAD

**Issue by floriandk** [maieul/ledmac#762]
# Version of reledmac / reledpar (if that is a development version, please precise the branch)
reledmac 2018/04/07 v2.26.7

# If this is a newly introduced problem, please indicate the last working version of the package(s)
2018/02/28 v2.26.6 worked correctly

# Does the problem happen only when combining with some packages?
no

# Does the problem happens both in parallel typesetting and normal typesetting?
yes

# Description of the problem
When using `\frontmatter` and `\mainmatter` the newer `reledmac`-version no longer correctly uses the mainmatter-pagenumber (1.26-2.2 in the MWE) but the total number (5.26-6.2).
Same goes for all pagenumbers when using endnotes.

(I noticed for the first time while experimenting with this problem:
Strictly speaking the formatting of the numbers should also be used by `reledmac`, i.e. when removing the line with `\mainmatter` from the MWE altogether, the note should read: "iv.26-v.2". But I have never been in a situation where this would be required yet. So I suppose it isn't important at all, but I thought I'd name it in case fixing the real bug would make it easy to also fix this minor one while you are at it.)

# Minimal Working Example
````latex
\documentclass[a5paper]{book}

\usepackage[series={A},noeledsec,nofamiliar,noledgroup]{reledmac}
\usepackage{reledpar}


%%same problem with endnotes:
%\let\Afootnote\Aendnote


\begin{document}
\frontmatter
one
\clearpage
two
\clearpage
three
\clearpage
\mainmatter
\lineation{page}
\firstlinenum*{1}
\linenumincrement*{1}

\beginnumbering
\pstart
Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi commodo, ipsum sed pharetra gravida, orci magna rhoncus neque, id pulvinar odio lorem non turpis. Nullam sit amet enim. Suspendisse id velit vitae ligula volutpat condimentum. Aliquam erat volutpat. Sed quis velit. Nulla facilisi. Nulla libero. Vivamus pharetra posuere sapien. Nam consectetuer. Sed aliquam, nunc eget euismod ullamcorper, lectus nunc ullamcorper orci, fermentum bibendum enim nibh eget ipsum. Donec porttitor ligula eu dolor. Maecenas vitae nulla consequat libero cursus venenatis. Nam magna enim, accumsan eu, blandit sed, blandit a, eros.

Quisque facilisis erat a dui. Nam malesuada ornare dolor. Cras gravida, diam sit amet rhoncus ornare, erat elit consectetuer erat, id egestas pede nibh eget odio. Proin tincidunt, velit vel porta elementum, magna diam molestie sapien, non aliquet massa pede eu diam. Aliquam iaculis. Fusce et ipsum et nulla tristique facilisis. Donec eget sem sit amet ligula viverra gravida. Etiam vehicula urna vel turpis. Suspendisse sagittis ante a urna. Morbi a est quis orci consequat rutrum. Nullam egestas feugiat felis. Integer adipiscing semper ligula. Nunc molestie, nisl sit amet cursus convallis, sapien lectus pretium metus, vitae pretium enim wisi id lectus. Donec vestibulum. Etiam vel nibh. Nulla facilisi. Mauris pharetra. Donec augue. Fusce ultrices, neque id dignissim ultrices, tellus mauris dictum elit, vel lacinia enim metus eu nunc.

Proin at eros non eros adipiscing mollis. %
\edtext{Donec semper turpis sed diam. Sed consequat ligula nec tortor. Integer eget sem. Ut vitae enim eu est vehicula gravida.}{%
	\Afootnote{del.}}
 Morbi ipsum ipsum, porta nec, tempor id, auctor vitae, purus. Pellentesque neque. Nulla luctus erat vitae libero. Integer nec enim. Phasellus aliquam enim et tortor. Quisque aliquet, quam elementum condimentum feugiat, tellus odio consectetuer wisi, vel nonummy sem neque in elit. Curabitur eleifend wisi iaculis ipsum. %
\edtext{Pellentesque}{%
	\Afootnote{del.}}
 habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. In non velit non ligula laoreet ultrices.
\pend
\endnumbering

\doendnotes{A}

\end{document}
````

**Comment by maieul** [comment]
Please try branch issue762

**Comment by floriandk** [comment]
Working as earlier both on testcase and real life example. Great!

(still not giving me Roman numerals when the apparatus is part of the frontmatter, but I suppose that you didn't intend to change this, did you?)

**Comment by maieul** [comment]
for roman numeral, it would be complex to do it automatically, because,
actually, we pass "raw" value of the page (we need it to do test).

What could be possible is to add setting to change page number
presentation in the footnote. You should open an issue for this.

**Comment by floriandk** [comment]
thanks for the explanation!

As I wrote above this is absolutely not a priority for me, I'll probably never use it.
By asking again I just wanted to make sure, that I wasn't supposed to test for this, too.
I'd say, if somebody really needs this feature one day, s/he'll find this discussion can open an issue at the time.

**Comment by floriandk** [comment]
Please reopen!

The changelog for 2.26.9 says that the mainmatter-bug should be fixed but it still has the faulty behaviour.

**Comment by maieul** [comment]
damned!

**Comment by maieul** [comment]
that is the first time I did a mistake in a merge commit ! Please try branch issue762_bis

**Comment by maieul** [comment]
or the branch 2.26.10-2.21.8 which merge it with branch issue763

**Comment by floriandk** [comment]
2.26.10-2.21.8 working fine for me. Thanks again!
